app/src/pages/inside/testCaseLibraryPage/allTestCasesPage/batchEditTagsModal/batchEditTagsModal.tsx

```
@@ -0,0 +1,199 @@
+  useEffect(() => {
+    getTags({ testCaseIds: selectedTestCaseIds })
+      .then((data) => {
+        setTagsList(data?.content);
+      })
+      .catch(noop);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const onRemoveTag = (tagKey: string): void => {
+    let tagToRemove: Tag;
+    const updatedTags = tagsList.filter((tag: Tag) => {
```

Biome (2.4.4)

[error] This callback passed to filter() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)